testing/runner.ts

```diff
@@ -12,26 +12,24 @@ const DEFAULT_GLOBS = [
   "**/test.js"
 ];
 
-/* eslint-disable max-len */
 function showHelp(): void {
   console.log(`Deno test runner
 
 USAGE:
   deno -A https://deno.land/std/testing/runner.ts [OPTIONS] [FILES...]
 
 OPTIONS:
-  -q, --quiet               Don't show output from test cases 
-  -f, --failfast            Stop test suite on first error
-  -e, --exclude <FILES...>  List of file names to exclude from run. If this options is 
-                            used files to match must be specified after "--". 
-  
+  -q, --quiet               Don't show output from test cases
+  -f, --failfast            Stop running tests on first error
+  -e, --exclude <FILES...>  List of comma-separated files to exclude from run
+
 ARGS:
-  [FILES...]  List of file names to run. Defaults to: ${DEFAULT_GLOBS.join(
-    ","
-  )} 
-`);
+  [FILES...]  List of files to run.
```

Comment:
Okay now I'm not sure, @nayeemrmn what will happen if I run test runner like that:
```
1. deno test src/
2. deno test src/my_test.ts
3. deno test foo/ bar/
4. deno test foo/test1.ts bar/test2.ts
5. deno test \*\*\/*_test.ts
```

Reply:
2, 4 and 5 would be the same as they are now,
1 would expand into `deno test src/**/test.ts src/**/*_test.ts` (also s/ts/js)
4 would expand into `deno test foo/**/test.ts foo/**/*_test.ts bar/**/test.ts bar/**/*_test.ts` (also s/ts/js ones)

It helps to execute the help message and see how that code evaluates.

Reply:
Okay, that's good. Then I suggest to update help message to better reflect that (possibly adding presented cases as examples).

`[FILES...]  List of files to run.` suggest that only file paths can be given but that's not true.

Reply:
> `[FILES...] List of files to run.` suggest that only file paths can be given but that's not true.

Yeah, that was bothering me. Forgot to change it with the first commit.

Reply:
I'll go with "List of test modules to run" like I have been in the code. "module" captures files, URLs, and directories according to Deno's nomenclature.